maquetado/setup/index.html

```diff
@@ -8,6 +8,15 @@
     <link rel="stylesheet"  type="text/css" href="application.css">    
   </head>
   <body>    
-      <h2 class="title">Front-End Tools</h2>      
+      <h2 class="title">Front-End Tools</h2>            
+      <div class="tech-container">
+        <div class="tech-back-container"></div>
+        <div class="tech-front-container">
+          <img class="tech-img" src="img/react_icon.svg">
+          <h1 class="tech-title">React</h1>
```

Comment:
`h1` element is for the big big title for the entire page, maybe an `h4` would fit more in this case